packages/aws_bedrock_agentcore/kibana/dashboard/aws_bedrock_agentcore-1f2ef1c3-afab-4b23-9e8b-b966f2fe40d2.json

```
@@ -0,0 +1,4531 @@
+                                "yTitle": "Latency"
+                            }
+                        },
+                        "title": "",
+                        "type": "lens",
+                        "visualizationType": "lnsXY"
+                    },
+                    "description": "Consolidation combines extracted memories with existing memories. Strategies determine what types of information to extract from raw conversations.",
+                    "enhancements": {
+                        "dynamicActions": {
+                            "events": []
+                        }
+                    },
+                    "filters": [],
+                    "query": {
+                        "language": "kuery",
+                        "query": ""
+                    },
+                    "syncColors": false,
+                    "syncCursor": true,
+                    "syncTooltips": false,
+                    "title": "Consolidation latency - by strategy type"
+                },
+                "gridData": {
+                    "h": 12,
+                    "i": "586189e4-e877-4bbb-b060-60adb54bddfc",
+                    "sectionId": "93a69d41-b068-4e1c-8fc6-ab16b1edfb2f",
+                    "w": 24,
+                    "x": 24,
+                    "y": 6
+                },
+                "panelIndex": "586189e4-e877-4bbb-b060-60adb54bddfc",
+                "type": "lens"
+            }
+        ],
+        "sections": [
+            {
+                "collapsed": false,
+                "gridData": {
+                    "i": "39c40c67-5aa1-43c4-995a-4415c9dda052",
+                    "y": 22
+                },
+                "title": "Extracted memory details"
+            },
+            {
+                "collapsed": false,
+                "gridData": {
+                    "i": "18f8eb6a-ed8c-4906-9de3-8ffa862bc2e2",
+                    "y": 23
+                },
+                "title": "Event details - CreateEvent, RetrieveMemoryRecords, ListEvents"
+            },
+            {
+                "collapsed": false,
+                "gridData": {
+                    "i": "e1538143-b3e1-493f-87f1-b1e8920bef64",
+                    "y": 24
+                },
+                "title": "Event details - List memory"
+            },
+            {
+                "collapsed": false,
+                "gridData": {
+                    "i": "490e6fcb-a864-4076-b1f5-dda7bac93d6a",
+                    "y": 25
+                },
+                "title": "Event details - Extractions"
+            },
+            {
+                "collapsed": false,
+                "gridData": {
+                    "i": "93a69d41-b068-4e1c-8fc6-ab16b1edfb2f",
+                    "y": 26
+                },
+                "title": "Event details - Consolidation"
+            }
+        ],
+        "timeRestore": false,
+        "title": "[Amazon Bedrock Agentcore] Memory",
```

nit, this applies to all dashboards

```suggestion
        "title": "[Amazon Bedrock AgentCore] Memory",
```

---

This will be taken up in the next change. Thanks for pointing out.